CONTRIBUTING.md

```diff
@@ -1,2 +1,13 @@
-# How to contribute
+# How to Contribute
+We welcome contributors to this repository! 
 
+If you find any error or if you would like to make a new feature, please create an issue to discuss the changes you would like to make. Any ideas and bug reports are encouraged!
+
+## Our Standards
+For us to foster an open and welcoming environment, please follow our standards below.
+* Use welcoming and inclusive language
+* Gracefully accept constructive criticism
+* Be respectful and focus on what is the best for the community
+
+## Attribution
+Our standards are adapted from the [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct), version 1.4, available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.
```

Comment:
` available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.`

This part is unnecessary since you already provided a link, feel free to remove.